src/Testcontainers.Xunit/ContainerFixture.cs

```diff
@@ -0,0 +1,73 @@
+namespace DotNet.Testcontainers.Xunit;
```

Comment:
```suggestion
namespace Testcontainers.Xunit;
```
I prefer the new namespace (I have not changed it for the Testcontainers project due to backward compatibility).

Reply:
Agreed! Addressed in d2669c01e4c7a952a70a3387723492b13a66b629.